src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -132,4 +257,25 @@ public String toAddressWithTab(Address address) {
         return claimantAddressStr.toString();
     }
 
+    private void populateCodeNameAndDescriptionHtml(StringBuilder sb, String codeName) {
```

Comment:
Whenever possible and practical, always get out early.  That is:
```
if(codeName==null) {
return;
}
...
```

This way there's less nesting which makes it easier on the eye